File: 283-MoveZeroes/283-MoveZeroes.py
# Last updated: 22/09/2025, 08:58:39
class Solution(object):
    def moveZeroes(self, nums):
        k=0
        for i in range(len(nums)):
            if nums[i]!=0:
                nums[i],nums[k]=nums[k],nums[i]
                k=k+1

File: 283-MoveZeroes/283-MoveZeroes.java
// Last updated: 22/09/2025, 08:58:36
class Solution {
    public void moveZeroes(int[] nums) {
        int k = 0;
        for(int i = 0; i<nums.length; i++){
            if (nums[i]==0) {
                continue;
            }
            nums[k++]=nums[i];
        }
        for(int i = k; i<nums.length; i++){
            nums[i] = 0;
        }
    }
}